python/lsst/meas/astrom/astrom.py

```diff
@@ -254,11 +259,11 @@ def useKnownWcs(self, sources, wcs=None, exposure=None, filterName=None, imageSi
         probably also want to turn OFF 'calculateSip'.
 
         This involves searching for reference sources within the WCS
-        area, and matching them to the given 'sources'.  If
+        area, and matching them to the given 'sourceCat'.  If
```

Comment:
For generic ordered containers I agree (and have continued to use �matches� instead of �matchList�), but I feel that tables act very differently than other ordered containers and some indication is in order, so I strongly prefer �sourceCat" to �sources�. To some extent this may be a C++ vs python thing; in C++ is often obvious what the type is by reading the header, but that is not true in Python. In trying to learn this code I have found many instances where the data types were not obvious and it really mattered.